src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -291,6 +297,14 @@ def load_command_table(self, _):
         g.custom_command('show-log', 'get_deployment_script_logs')
         g.custom_command('delete', 'delete_deployment_script', confirmation=True)
 
+    with self.command_group('ts', resource_templatespecs_sdk, resource_type=PROFILE_TYPE, is_preview=True, min_api='2019-06-01-preview') as g:
```

Comment:
What is PROFILE_TYPE?
Should it be ResourceType.MGMT_RESOURCE_TEMPLATESPECS?

Reply:
To add the min_api as requested requires changing the resourcec_type to PROFILE_TYPE otherwise the tests for the profiles that are not latest fail because MGMT_RESOURCE_TEMPLATESPECS does not exist and the api version cannot be found.

Reply:
@detienne20 May I ask if this command can only appear in the latest profile?
If not, you can add the same version config of the latest profile to other profiles, and then change the `resource_type=PROFILE_TYPE` to `resource_type=ResourceType.MGMT_RESOURCE_TEMPLATESPECS`

Reply:
This command can only appear in the latest profile, resources api needs to be 2020-06 at the very least, that is when template-specs is added.

Reply:
@detienne20 I still suggest that you set a default version in other profiles before `2019-06-01-preview`, so that the judgment logic of `min_api` is based on `ResourceType.MGMT_RESOURCE_TEMPLATESPECS`.

Reply:
Done.